src/elements/ProgressBar/ProgressBar.styl

```diff
@@ -84,13 +84,18 @@
 
     border none
     border-radius var(--progress-bar-border-radius)
+    color var(--progress-bar-bar-color)
 
     overflow hidden
 
     // States
 
     &[value]
-      &::-moz-progress-bar,
+      &::-moz-progress-bar
+        background-color var(--progress-bar-bar-color)
+
+        transition width $duration-shortest
```

Comment:
Can this same transition also be applied to `-webkit-progress-value` below?

Reply:
It is already applied to `-webkit-progress-value`. Please reconfirm